### DISCUSSION TITLE
Dumb question

### DISCUSSION THREAD

**Discussion by J4k-The-Reaper** [DIGITALCRIMINALS/OnlyFans#412] [General]
So I've always been curious.....as I have four 2 terabyte external hard drives full of scrapes only so far....and counting. But why do us guys-and, hey, gals too, don't want to be sexist and all- scrape so much and save it? I mean....there's plenty of pron out there. Why are we so attached to saving onlyfans content? It's not like I'll watch or look at every scrape!! Haha. I dunno....it's just an observation I've had for awhile but was nervous to ask...did not want to make it seem judgy or inappropriate asking. It's an honest question and curious thought.

**Comment by MonkeyKingViper** [comment]
Meh. I'm a collector of trophies. Nudes of women I know and have been with, nudes of women I don't and never will. Leaks. OF. IG. All the stuff. I think that's what it is for me. This model posts a vid of her doing something. When she removes it later, it's gone forever unless someone happened to save it. I have tons of models that I've saved their stuff for a year, then decided to redo my file naming and find they've culled a thousand images from their page. It's possible I have the only copies of those that exist. 

I know one woman that did an OF for about 3 months before someone figured out who she was and threatened her till she disappeared. I guarantee I have content of her that is one of a kind. I look at it periodically. She was great. Shame.

Another model did some NN but very revealing shoots with some photogs right after she turned 18. 3 years later she regrets them, and tried to scrub them from the internet. I have some of them, and are looking for more.

A woman I knew took a few modeling photos showing her pierced nips and put them on tumblr in 2010. They are the only ones in existence as she's a good Christian girl. I have them.

Another model I know and chill with IRL did nude modeling 10 years ago. She was real popular on "The Dirty". I have all her sets and have shared them a few places.

So yeah, women are like baseball cards to me. They're trophies and also commodities. I can put a PPV for 20-30 and then share it on one of the major websites and get praise to stroke my ego, and some rando will post a 20-30 ppv of a model I want to see. That's why sharing is caring. Everyone profits.
